spec/cypress/integration/Cookies.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Cookie page
+  All ECF users needs to be able to set and modify their cookie preferences
+
+  Scenario: Preferences should not be initially set
+    Given I am on "cookie" page
+    Then "cookie consent" radios are unchecked
+
+  Scenario: Setting cookie preferences on cookie page
+    Given I am on "cookie" page
+
```

Comment:
Delete empty space

Reply:
This empty space was specifically requested by @ethanmills 😆